vnext/ReactWindowsCore/WinRTWebSocketResource.h

```diff
@@ -54,13 +56,20 @@ class WinRTWebSocketResource : public IWebSocketResource, public std::enable_sha
 
   void OnMessageReceived(
       winrt::Windows::Networking::Sockets::IWebSocket const &sender,
-      winrt::Windows::Networking::Sockets::MessageWebSocketMessageReceivedEventArgs const &args);
+      winrt::Windows::Networking::Sockets::IMessageWebSocketMessageReceivedEventArgs const &args);
   void Synchronize();
 
  public:
+  WinRTWebSocketResource(
+      winrt::Windows::Networking::Sockets::IMessageWebSocket &&socket,
+      winrt::Windows::Storage::Streams::IDataWriter &&writer,
+      winrt::Windows::Foundation::Uri &&uri,
+      std::vector<winrt::Windows::Security::Cryptography::Certificates::ChainValidationResult> certExeptions);
```

Comment:
>vector [](start = 11, length = 6)

This should be &&

Reply:
Same for line 71

---
In reply to: [418654423](https://github.com/microsoft/react-native-windows/pull/4767#discussion_r418654423) [](ancestors = 418654423)